OSDOCS-13516-v415 switched steps 7 and 8 under Restart the Cluster (#90230)
openshift/openshift-docs: modules/graceful-restart.adoc

$ oc adm certificate approve <csr_name>
----

> . After the control plane and worker nodes are ready, mark all the nodes in the cluster as schedulable by running the following command:

Reviewer: ```suggestion
. After the control plane and compute nodes are ready, mark all the nodes in the cluster as schedulable by running the following command:
```

$ oc adm certificate approve <csr_name>
----

> . After the control plane and worker nodes are ready, mark all the nodes in the cluster as schedulable.
> Run the following command:

Reviewer: ```suggestion
. After the control plane and compute nodes are ready, mark all the nodes in the cluster as schedulable by running the following command:
```

+
[source,terminal]
----

> for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do echo ${node} ; oc adm uncordon ${node} ; done

Reviewer: ```suggestion
$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do echo ${node} ; oc adm uncordon ${node} ; done
```
